webapi/src/main/scala/dsp/valueobjects/LanguageCode.scala

```diff
@@ -43,14 +44,14 @@ object LanguageCode { self =>
     } else if (!SupportedLanguageCodes.contains(value)) {
       Validation.fail(ValidationException(LanguageCodeErrorMessages.LanguageCodeInvalid(value)))
     } else {
-      Validation.succeed(new LanguageCode(value) {})
+      Validation.succeed(LanguageCode(value))
     }
 
-  lazy val en: LanguageCode = new LanguageCode(EN) {}
-  lazy val de: LanguageCode = new LanguageCode(DE) {}
-  lazy val fr: LanguageCode = new LanguageCode(FR) {}
-  lazy val it: LanguageCode = new LanguageCode(IT) {}
-  lazy val rm: LanguageCode = new LanguageCode(RM) {}
+  lazy val en: LanguageCode = LanguageCode(EN)
+  lazy val de: LanguageCode = LanguageCode(DE)
+  lazy val fr: LanguageCode = LanguageCode(FR)
+  lazy val it: LanguageCode = LanguageCode(IT)
+  lazy val rm: LanguageCode = LanguageCode(RM)
```

Comment:
IMO type could be removed here too.

Reply:
This is just code I moved, also sometimes I like explicit type declarations and would like to leave any discussions about how to write type declarations out of the discussion for any PR.